src/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { StatelessComponent, useState, useEffect } from 'react';
+import { track } from './utils';
+
+const Footer: StatelessComponent = () => {
+  const [expanded, setExpanded] = useState(false);
+
+  useEffect(() => {
+    track('footer-render');
+  });
+
+  return (
+    <legend>
+      <hr />
+      Copyright Aperture Science Laboratories
+      <span onClick={() => setExpanded(!expanded)}>🍰</span>
```

Comment:
a11y - span with onClick is not accessible without additional markup... or just make it a button!